src/styles.css

```diff
@@ -54,6 +55,14 @@ p {
     padding: 100px 0;
 }
 
+header {
+    text-align: center;
+    border-radius: 4px;
+    border: 4px black;
+    height: 1000px;
+}
+
+
 .landkaart {
```

Comment:
Een tip: gebruik alleen engelse class names of 1 taal voor alle classes en ID's :)

Reply:
inderdaad!